src/ActionButtons/index.js

```diff
@@ -9,37 +9,49 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
-import React from 'react'
```

Comment:
I am not familiar with `@emotion`. What is the benefit of it? 
IMHO, having simple React makes the code easier to read, maintain and more solid

Reply:
@emotion allows us to do CSS in JS a lot easier.